### PR TITLE
T62 add product window

### PR DESCRIPTION
**Pull request by BirhanBey**
added new item window

### CODE REVIEW

**Comment by BirhanBey** [comment]
[create add product window](https://trello.com/c/FtKBfxvu/62-create-add-product-window)

File: admin/index.html [hunk 1]
```diff
@@ -89,11 +89,107 @@ <h1>Products</h1>
...
     <!-- New Item form -->
-    <div></div>
+    <div class="new-item">
+      <div class="background">
+      <p class="border"></p>
+      <div class="form">
+        <div class="info info1">
+          <i class="fa-solid fa-hashtag icon"></i>
+          <label for="product-id">ID</label><br />
+          <input
```
Comment (CCornez) [inline admin/index.html:104]: this shouldn't be an input, id can not be change

File: admin/index.html [hunk 2]
```diff
@@ -89,11 +89,107 @@ <h1>Products</h1>
...
+            type="text"
+            id="product-id"
+            name="fav_language"
+            placeholder="  ID"
+          />
+        </div>
+        <div class="info info2">
+          <i class="fa-regular fa-image icon"></i>
+          <label class="active" for="image">Image</label><br />
```
Comment (CCornez) [inline admin/index.html:113]: why is there an active class ?

File: admin/index.html [hunk 3]
```diff
@@ -115,5 +211,35 @@ <h1>Products</h1>
       }
       list.forEach((item) => item.addEventListener("click", activeLink));
     </script>
+
+    <!--  -->
+    <!-- add new item focus-- this should move when you click each input but ony first one is working Ihope you can fix it or we can just delete it-->
```
Comment (CCornez) [inline admin/index.html:216]: Looks really good! I'll change it so it works on the other items

**Review by CCornez — CHANGES_REQUESTED** [review]

**Review by CCornez — APPROVED** [review]
PERFECT

**Pull request merged**